qrcode/main.py

```diff
@@ -76,7 +89,10 @@ def make(self, fit=True):
         """
         if fit or (self.version is None):
             self.best_fit(start=self.version)
-        self.makeImpl(False, self.best_mask_pattern())
+        if self.mask_pattern == None:
```

Comment:
`is None` (or you could just do a single `self.mask_pattern if self.mask_pattern else self.best_mask_pattern()`)